README.md

```diff
@@ -78,6 +78,7 @@ the following map to the given option
 - `VINNIE_GITLAB_TOKEN` sets `--gitlab-token`
 - `VINNIE_REPO_PUSH` sets `--push`
 - `VINNIE_PREFIX` sets `--prefix`
+- `VINNIE_OMIT_PREFIX` sets `--omit-prefix`
```

Comment:
We'll need to also document what this options does.

Reply:
https://github.com/revsys/vinnie/pull/9/files#diff-04c6e90faac2675aa89e2176d2eec7d8R62